cmake_modules/commonSetup.cmake

```diff
@@ -839,10 +839,8 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
         endif()
       endif(USE_URIPARSER)
 
-      find_package(Boost COMPONENTS regex)
-	  if (NOT Boost_REGEX_FOUND)
-		message(FATAL_ERROR "BOOST_REGEX required but package not found")
-	  endif()
+      # Do we need this at the top level or just within the components that use it?
+      # find_package(pcre2 CONFIG REQUIRED)
```

Comment:
Its preferable to have it just in the components that need it (IMO)

Reply:
I will remove that comment.  (It is actually used within rtl/eclrtl/CMakeLists.txt).